bbcode-src/tags/block.js

```diff
@@ -7,7 +7,7 @@ import { preprocessAttr } from "../utils/common";
  * @example [block=treasure]content[/block]
  */
 export const block = (node) => {
-    const block = preprocessAttr(node.attrs)._default || "block";
+    const blockAttr = preprocessAttr(node.attrs)._default.toLowerCase() || "block";
```

Comment:
`_default` can be undefined. toss an optional chain before the `toLowerCase()`

Reply:
Gotcha. Tested against option not existing and it passed so assumed process enforced string even if undefined.  Will fix!